test_runner/src/main/kotlin/ftl/run/model/AndroidTestContext.kt

```diff
@@ -2,10 +2,15 @@ package ftl.run.model
 
 import ftl.api.FileReference
 import ftl.api.ShardChunks
+import ftl.args.AndroidArgs
 import ftl.args.IgnoredTestCases
 import ftl.shard.Chunk
 
-sealed class AndroidTestContext
+interface WithArgs {
```

Comment:
Why an interface at this point instead of simply:

```suggestion
sealed class AndroidTestContext(open val args: AndroidArgs)
```

Reply:
Hm...why not, I do not have a strong opinion about it 👍🏻